### ISSUE TITLE
can't clone this repo

### ISSUE THREAD

**Issue by SurgeonY** [linanqiu/word2vec-sentiments#17]
Error downloading object: train-unsup.txt (6539951): Smudge error: Error downloading train-unsup.txt (6539951b1bfd1420c2df99fe824272dd846cd25bca4a2247a4266f12aec9f195): batch response: This repository is over its data quota. Purchase more data packs to restore access.

This repository is over its data quota. Purchase more data packs to restore access.
github.com/git-lfs/git-lfs/errors.newWrappedError
        C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/errors/types.go:170: batch response
github.com/git-lfs/git-lfs/errors.newWrappedError
        C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/errors/types.go:170: Error downloading train-unsup.txt (6539951b1bfd1420c2df99fe824272dd846cd25bca4a2247a4266f12aec9f195)
github.com/git-lfs/git-lfs/errors.newWrappedError
        C:/Users/ttaylorr/go/src/github.com/git-lfs/git-lfs/src/github.com/git-lfs/git-lfs/errors/types.go:170: Smudge error

**Comment by linanqiu** [comment]
Yeah I need to move the hosting to somewhere else. :(

**Comment by samir-ware** [comment]
First and foremost, thank you so much for amazing tutorial. I followed how words are arranged semantically in word2Vec although can you please elaborate a bit more on how "exactly" user sentiments are captured in this code ? 

If you could give a specific example as to how to get sentiment of some data it will be a great help. I can see that there is a file with unsuperwised data which never got used. I am hoping this code will somehow understand the sentiments in that data too although not sure how. :( 

Also I could not see labeledTrainData and in labeledTrainData files anywhere in this repo .
Can you please add those too ?

Thanks
Samir

**Comment by SurgeonY** [comment]
Cool tutorial though, thanks for this! You'd won actually kaggle [competition](https://www.kaggle.com/c/sentiment-analysis-on-movie-reviews) with your 0.87 accuracy! The leader board had only 0.76 at closing :)

**Comment by transfluxus** [comment]
can you provide the `train-unsup.txt` somewhere

**Comment by SurgeonY** [comment]
transfluxus, You can preprocess it easily on your own from the original review files, download them from here http://ai.stanford.edu/~amaas/data/sentiment/
It's easy or just take my script, in my fork https://github.com/SurgeonY/word2vec-sentiments 
 prepare-texts.py
Cheers

**Comment by pingdanEYRE** [comment]
@SurgeonY Excuse me do you have a 0.87 accuracy rate? I have only 0.64 accuracy.Thank you for your answer

**Comment by SurgeonY** [comment]
@pingdanEYRE Yes, why not?)) Even with a bit of playing with params but anyway, all around~87%, check my forked repo.
<img width="707" alt="87" src="https://user-images.githubusercontent.com/500791/35783081-892b4a80-0a0a-11e8-867b-19c6cfc7657f.png">

Though I agree with this guy https://github.com/linanqiu/word2vec-sentiments/issues/21, we should not train Doc2Vec on testing set, instead should infer testing doc vectors right at the moment of scoring/testing. This could lower the score, but yet I have to check it out.
